app/models/comment.rb

```diff
@@ -0,0 +1,10 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_post_comments_counter
+
```

Comment:
Great job @RaphDasilva 🤝

- [ ] [Optional] You may also make the `update_post_comments_counter` method private.